l10n_br_sale/models/sale.py

```diff
@@ -130,252 +129,100 @@ def _default_fiscal_category(self):
     )
```

Comment:
When porting to v12, I propose to let the user use instead the generic OCA module https://github.com/OCA/sale-workflow/tree/12.0/sale_order_general_discount instead of this discount_rate instead. As this module doesn't exist in v10, okay to keep it like this in v10.